README.md

```diff
@@ -1392,6 +1392,7 @@ _See [Ticketing](#ticketing)_
 
 ### IDE/Tools
 
+- [Appsmith](https://www.appsmith.com/) - Cloud or self-hosted open-source platform to build admin panels, CRUD apps and workflows. Build everything you need, 10x faster. ([Demo](https://www.youtube.com/watch?v=mzqK0QIZRLs), [Source Code](https://github.com/appsmithorg/appsmith)) `Apache-2.0` `Java/Docker`
```

Comment:
The Demo link is only for interactive demos, a video is documentation, please remove the demo link.

Sorry, I think this we should make this more clear in https://github.com/awesome-selfhosted/awesome-selfhosted/blob/master/.github/PULL_REQUEST_TEMPLATE.md - i will open an issue.

Reply:
Hey @nodiscc, I've updated the PR :)